docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -0,0 +1,352 @@
+===============================
+GlowGR: Whole-Genome Regression
+===============================
+
+.. invisible-code-block: python
+
+    import glow
+    glow.register(spark)
+
+    genotypes_vcf = 'test-data/gwas/genotypes.vcf.gz'
+    covariates_csv = 'test-data/gwas/covariates.csv.gz'
+    continuous_phenotypes_csv = 'test-data/gwas/continuous-phenotypes.csv.gz'
+
+Glow supports Whole Genome Regression (WGR) as GlowGR, a parallelized version of the
+`regenie <https://rgcgithub.github.io/regenie/>`_ method (see the
+`preprint <https://www.biorxiv.org/content/10.1101/2020.06.19.162354v1>`_).
+
+.. image:: ../_static/images/wgr_runtime.png
+   :scale: 50 %
+
+GlowGR consists of the following stages.
+
+- Blocking the genotype matrix across samples and variants.
+- Performing dimension reduction with ridge regression.
+- Estimating phenotypic values with ridge regression.
+
+----------------
+Data preparation
+----------------
+
+GlowGR uses three basic input datasources.
+
+Genotype data
+=============
+
+The genotype data may be read from any variant datasource supported by Glow, such as VCF, BGEN or PLINK. The DataFrame
+must also include a column ``values`` containing a numeric representation of each genotype. The genotypic values may
+not be missing, or equal for every sample in a variant.
```

Comment:
what does equal mean here? All homozygous reference?

Reply:
Mathematically, we're trying to filter out variants for which all samples have the same calls and therefore `values` has a variance/stddev of `0` (eg. 
all hom ref, all hom-alt, or even all het). I'm not sure what the best way to phrase this is.